lms/envs/common.py

```diff
@@ -2870,3 +2870,14 @@
 TYPES_WITH_CHILD_PROBLEMS_LIST = [
     'library_content',
 ]
+
+# These are the problem types (by block_type) that an instructor can query over in the Instructor Query tab
+INSTRUCTOR_QUERY_PROBLEM_TYPES = [
```

Comment:
@stvstnfrd is there a particular place where we insert this kind of thing? or is the list of vars in aws.py more or less random?